docs/user/commandline_tools.rst

```diff
@@ -32,6 +31,10 @@ Run ``geomesa`` without any arguments to produce the following usage text::
         ingestraster        Ingest a raster file or raster files in a directory into GeoMesa
         list                List GeoMesa features for a given catalog
         queryrasterstats    Export queries and statistics about the last X number of queries to a CSV file.
+        stats-analyze       Analyze statistics on a GeoMesa feature type
+        stats-bounds        View bounds on attributes in a GeoMesa schema
+        stats-count         View feature counts in a GeoMesa schema
+        stats-histogram     View statistics on a GeoMesa feature type
```

Comment:
[picky] improve description for stats-histogram. Also make the descriptions more consistent (i.e. "on a GeoMesa feature type" vs "in a GeoMesa schema")